### PR TITLE
Add new DVC extra

### PR DESCRIPTION
**Pull request by mike0sv**

### CODE REVIEW

**Review by aguschin — APPROVED** [review]

File: setup.py [hunk 1]
```diff
@@ -110,6 +110,7 @@
     "webdav",
     "webhdfs",
     "webhdfs_kerberos",
+    "lint",
```
Comment (skshetry) [inline setup.py:113]: Why is this needed? We usually move these things around without a guarantee that they will always work or be available (other than remote related extras).

Comment (aguschin) [reply setup.py:113]: This is to enable `pip install mlem[dvc-s3]` and ones like that. We have a test that checks we didn't miss anything, that's why we need to add keep it here.

Comment (aguschin) [reply setup.py:113]: btw, I think we should've added it here instead https://github.com/iterative/mlem/blob/d2100acaa8b092bc416cfc65ab0583a90b2e776d/tests/test_setup.py#L18

Comment (aguschin) [reply setup.py:113]: doing that in https://github.com/iterative/mlem/pull/634
